include/openssl/aes.h

```diff
@@ -73,17 +73,16 @@ void AES_cfb8_encrypt(const unsigned char *in, unsigned char *out,
 void AES_ofb128_encrypt(const unsigned char *in, unsigned char *out,
                         size_t length, const AES_KEY *key,
                         unsigned char *ivec, int *num);
-# ifndef OPENSSL_NO_DEPRECATED_3_0
+
 /* NB: the IV is _two_ blocks long */
-void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
-                     size_t length, const AES_KEY *key,
-                     unsigned char *ivec, const int enc);
+DEPRECATEDIN_3_0(void AES_ige_encrypt(const unsigned char *in,
+    unsigned char *out, size_t length, const AES_KEY *key, unsigned char *ivec,
+    const int enc))
 /* NB: the IV is _four_ blocks long */
-void AES_bi_ige_encrypt(const unsigned char *in, unsigned char *out,
-                        size_t length, const AES_KEY *key,
-                        const AES_KEY *key2, const unsigned char *ivec,
-                        const int enc);
-# endif
+DEPRECATEDIN_3_0(void AES_bi_ige_encrypt(
+    const unsigned char *in, unsigned char *out, size_t length,
+    const AES_KEY *key, const AES_KEY *key2, const unsigned char *ivec,
+    const int enc))
```

Comment:
Is this alignment according to code style?  I'm sure @DDvO's checker script is gonna say a few things about these lines.

(and yes, I'm perfectly aware how very ugly the alternative is.  Still...)

Reply:
> Is this alignment according to code style? I'm sure @DDvO's checker script is gonna say a few things about these lines.
> 
> (and yes, I'm perfectly aware how very ugly the alternative is. Still...)

Indeed, `check-format.pl` complains on that decl and the one before like this:
```
aes.h:79:indent = 4 != 38 for hanging '(':    unsigned char *out, size_t length, const AES_KEY *key, unsigned char *ivec,
aes.h:80:indent = 4 != 38 for hanging '(':    const int enc))
aes.h:83:indent = 4 != 41 for hanging '(':    const unsigned char *in, unsigned char *out, size_t length,
aes.h:84:indent = 4 != 41 for hanging '(':    const AES_KEY *key, const AES_KEY *key2, const unsigned char *ivec,
aes.h:85:indent = 4 != 41 for hanging '(':    const int enc))
```

A viable alternative formatting could be:
```
/* NB: the IV is _two_ blocks long */
DEPRECATEDIN_3_0(void
                 AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                                 size_t length, const AES_KEY *key,
                                 unsigned char *ivec, const int enc))
/* NB: the IV is _four_ blocks long */
DEPRECATEDIN_3_0(void
                 AES_bi_ige_encrypt(const unsigned char *in, unsigned char *out,
                                    size_t length, const AES_KEY *key,
                                    const AES_KEY *key2,
                                    const unsigned char *ivec, const int enc))
```

Reply:
Does this work?
```
DEPRECATEDIN_3_0(
    void
    AES_bi_ige_encrypt(const unsigned char *in, unsigned char *out, size_t length,
                       const AES_KEY *key, const AES_KEY *key2,
                       const unsigned char *ivec, const int enc)
)
```
And ```const int``` is pointless BTW.

Reply:
The tool does not like that, neither does the Emacs C mode.

I've already added the possibility for such a relaxation in the tool the allows for alternative indentation like this within expressions, but not (yet?) at statement/declaration level.